angular-material/angular-material.d.ts

```diff
@@ -73,6 +73,12 @@ declare namespace angular.material {
         initialValue(initialValue: string): IPromptDialog;
     }
 
+    interface IColorService {
+        applyThemeColors(element:Element|JQuery, colorExpression: Object): void;
```

Comment:
According to the docs, `colorExpression` is a `json object, keys are css properties and values are string of the wanted color, for example: {color: 'red-A200-0.3'}.` The type could be `{ [cssPropertyName: string]: string }`.